src/applications/disability-benefits/686c-674/analytics/helpers.js

```diff
@@ -0,0 +1,22 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+export const fireAnalytics = submission => {
+  recordEvent({
+    event: 'disability-21-686c-submission-successful',
+    'disability-under18AndUnmarried':
+      submission['view:selectable686Options'].reportMarriageOfChildUnder18,
+    'disability-childAttendingSchool':
+      submission['view:selectable686Options']
+        .reportChild18OrOlderIsNotAttendingSchool,
```

Comment:
Should this key correspond to the 674 workflow?

Reply:
Updated this